_extras/checkout.md

```diff
@@ -247,10 +247,11 @@ also email you to let you know, as this information is typically not shared duri
 after you have attended your teaching demonstration.
 
 ## 4. Tracking Your Progress
-After you complete this workshop, your application will be matched to your training record and a profile will be created in The
+After your instructor training workshop workshop, Carpentries Core Team members will activate
```

Comment:
```suggestion
After your instructor training workshop workshop, Carpentries Core Team members will use your application or profile-creation form to activate
```

Reply:
I'm making the use of the form more explicit because failure to submit the form is often a sticking point, so it's good to have one more place to remind them that it is necessary.

Reply:
is it choose once application or profile creation form, or is that two names for the same form. 
If someone has not done this, how would they know? how can a trainer help them if they ask questions about that in a workshop?